sirepo/package_data/static/js/radia.js

```diff
@@ -1343,6 +1349,144 @@ SIREPO.app.directive('bevelTable', function(appState, panelState, radiaService)
     };
 });
 
+SIREPO.app.directive('filletTable', function(appState, panelState, radiaService) {
+    return {
+        restrict: 'A',
+        scope: {
+            field: '=',
+            fieldName: '=',
+            itemClass: '@',
+            model: '=',
+            modelName: '=',
+            parentController: '=',
+            object: '=',
+        },
+        template: `
+            <table class="table table-hover">
+              <colgroup>
+                <col style="width: 20ex">
+                <col style="width: 20ex">
+                <col style="width: 20ex">
+              </colgroup>
+              <thead>
+                <tr>
+                  <th>Axis</th>
+                  <th>Edge</th>
+                  <th>Radius</th>
+                  <th>Resolution</th>
+                  <th></th>
+                </tr>
+              </thead>
+             <tbody>
+            <tr>
+            </tr>
+                <tr data-ng-repeat="item in loadItems()">
+                    <td>{{ item.cutAxis }}</td>
+                    <td>{{ bevelEdge(item.edge) }}</td>
+                    <td>{{ item.radius }}mm</td>
+                    <td>{{ item.numSides }}</td>
+                  <td style="text-align: right">
+                    <div class="sr-button-bar-parent">
+                        <div class="sr-button-bar" data-ng-class="sr-button-bar-active" >
+                             <button data-ng-click="editItem(item)" class="btn btn-info btn-xs sr-hover-button">Edit</button>
+                             <button data-ng-click="deleteItem(item, $index)" class="btn btn-danger btn-xs"><span class="glyphicon glyphicon-remove"></span></button>
+                        </div>
+                    <div>
```

Comment:
Should be `</div>`, also applies to L1254.

Reply:
[681e384](https://github.com/radiasoft/sirepo/pull/4699/commits/681e384af5430814034d4778780c0f6a0ff5ab39)